src/actionCanRead.cpp

```diff
@@ -116,7 +120,7 @@ execution ActionCanRead::execute(const TestRepetitions&)
         std::cout << std::endl << "Failed to receive data over the CAN bus !!" << std::endl << std::endl;
     }
     
-    driver_.close();
+ //   pdr_ptr->close();
```

Comment:
The removal of the `close()` is intentional

Reply:
openPolyDriver will open/close the driver  cc @triccyx